### PR TITLE
Feat: implement input select at workspace invite

### PR DESCRIPTION
**Pull request by itsmdasifraza**
## Description
implement input select at workspace invite
Fixes issue #194 

## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build

## Have you tested locally?

- [x] 👍 yes
- [ ] 🙅 no, because I am lazy

### CODE REVIEW

File: src/lib/components/inputs/InputSelect.svelte [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+  .select-data {
+    background-color: rgba(0, 0, 0, 0.7);
+    color: white;
+    position: absolute;
+    bottom: 0px;
+    left: 0;
+    right: 0;
+    transform: translateY(100%);
+    border: 1px solid rgb(44, 44, 44);
+    max-height: 200px;
```
Comment (LordNayan) [inline src/lib/components/inputs/InputSelect.svelte:218]: Can we use calc here

File: src/lib/components/inputs/InputSelect.svelte [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+    overflow-y: auto;
+    -webkit-backdrop-filter: blur(10px); /* For some older versions of Safari */
+    backdrop-filter: blur(10px);
+  }
+  .select-data p {
+    font-size: 12px;
```
Comment (LordNayan) [inline src/lib/components/inputs/InputSelect.svelte:224]: I think we have previously used fs class to set 12 px. Lets use that.

File: src/lib/components/inputs/InputSelect.svelte [hunk 3]
```diff
@@ -0,0 +1,249 @@
...
+    font-weight: 400;
+  }
+  .select-active {
```
Comment (LordNayan) [inline src/lib/components/inputs/InputSelect.svelte:227]: Can use d-block

File: src/lib/components/inputs/InputSelect.svelte [hunk 4]
```diff
@@ -0,0 +1,249 @@
...
+
+<style>
+  .email-container {
+    display: flex;
+    flex-wrap: wrap;
+    background-color: transparent;
+    border: 1px solid;
+    padding: 3px 8px 3px 8px;
+    border: 1px solid var(--border-color);
+    max-height: 100px;
```
Comment (LordNayan) [inline src/lib/components/inputs/InputSelect.svelte:196]: calc

File: src/lib/components/inputs/InputSelect.svelte [hunk 5]
```diff
@@ -0,0 +1,249 @@
...
+          </p>
+        {/if}
+      </div>
+    {/if}
+  </div>
+</div>
+
+<style>
+  .email-container {
+    display: flex;
```
Comment (LordNayan) [inline src/lib/components/inputs/InputSelect.svelte:190]: bootstrap classes

File: src/lib/components/inputs/InputSelect.svelte [hunk 6]
```diff
@@ -0,0 +1,249 @@
+<script lang="ts">
+  import dropdown from "$lib/assets/dropdown.svg";
+  import checkIcon from "$lib/assets/check.svg";
+  import closeIcon from "$lib/assets/close-icon-normal.svg";
+  import closeIconWhite from "$lib/assets/close-icon-white.svg";
+  import { createDynamicComponents } from "$lib/utils/helpers/common.helper";
+  import { validateEmail } from "$lib/utils/helpers";
+  import { onDestroy, onMount } from "svelte";
+  import { fade, fly, slide } from "svelte/transition";
+  let isOpen: boolean = false;
```
Comment (LordNayan) [inline src/lib/components/inputs/InputSelect.svelte:10]: Can remove type from here

File: src/lib/components/inputs/InputSelect.svelte [hunk 7]
```diff
@@ -0,0 +1,249 @@
...
+  export let isError: boolean = false;
```
Comment (LordNayan) [inline src/lib/components/inputs/InputSelect.svelte:11]: From here as well

File: src/lib/components/inputs/InputSelect.svelte [hunk 8]
```diff
@@ -0,0 +1,249 @@
...
+    padding: 3px 8px 3px 8px;
+    border: 1px solid var(--border-color);
+    max-height: 100px;
+    overflow-y: auto;
+  }
+  input {
+    margin-bottom: 4px !important;
+    margin-top: 4px !important;
+  }
+  #input-email {
```
[the remark this answers is not in the document]
Comment (itsmdasifraza) [reply src/lib/components/inputs/InputSelect.svelte:203]: resolved

File: src/lib/components/inputs/InputSelect.svelte (hunk 1 above)
Comment (itsmdasifraza) [reply src/lib/components/inputs/InputSelect.svelte:218]: resolved

File: src/lib/components/inputs/InputSelect.svelte [hunk 9]
```diff
@@ -0,0 +1,249 @@
...
+  }
+  .select-active {
+    display: block !important;
+  }
+  .highlight {
+    border-radius: 4px;
+    cursor: pointer;
+  }
+  .highlight:hover {
+    background-color: #232424;
```
[the remark this answers is not in the document]
Comment (itsmdasifraza) [reply src/lib/components/inputs/InputSelect.svelte:235]: resolved

File: src/lib/components/inputs/InputSelect.svelte (hunk 2 above)
Comment (itsmdasifraza) [reply src/lib/components/inputs/InputSelect.svelte:224]: resolved

File: src/lib/components/inputs/InputSelect.svelte (hunk 3 above)
Comment (itsmdasifraza) [reply src/lib/components/inputs/InputSelect.svelte:227]: resolved

File: src/lib/components/inputs/InputSelect.svelte (hunk 4 above)
Comment (itsmdasifraza) [reply src/lib/components/inputs/InputSelect.svelte:196]: resolved

File: src/lib/components/inputs/InputSelect.svelte (hunk 5 above)
Comment (itsmdasifraza) [reply src/lib/components/inputs/InputSelect.svelte:190]: resolved

File: src/lib/components/inputs/InputSelect.svelte (hunk 6 above)
Comment (itsmdasifraza) [reply src/lib/components/inputs/InputSelect.svelte:10]: resolved

File: src/lib/components/inputs/InputSelect.svelte (hunk 7 above)
Comment (itsmdasifraza) [reply src/lib/components/inputs/InputSelect.svelte:11]: resolved

**Review by LordNayan — APPROVED** [review]

**Pull request merged**